addons/website/static/tests/mock_server/mock_models/website_visitor.js

```diff
@@ -19,76 +18,4 @@ export class WebsiteVisitor extends models.ServerModel {
                 `Website Visitor #${record.id}`;
         }
     }
-
-    /** @param {number[]} ids */
-    _to_store(ids, store) {
-        /** @type {import("mock_models").ResCountry} */
-        const ResCountry = this.env["res.country"];
-        /** @type {import("mock_models").ResLang} */
-        const ResLang = this.env["res.lang"];
-        /** @type {import("mock_models").ResPartner} */
-        const ResPartner = this.env["res.partner"];
-        /** @type {import("mock_models").Website} */
-        const Website = this.env["website"];
-
-        for (const visitor of this.browse(ids)) {
-            const [data] = this._read_format(visitor.id, ["display_name"]);
-            data.country_id = mailDataHelpers.Store.one(ResCountry.browse(visitor.country_id));
-            data.history = visitor.history;
-            data.lang_id = mailDataHelpers.Store.one(ResLang.browse(visitor.lang_id));
-            data.partner_id = mailDataHelpers.Store.one(
-                ResPartner.browse(visitor.partner_id),
-                makeKwArgs({ fields: ["country_id"] })
-            );
-            data.website_id = mailDataHelpers.Store.one(Website.browse(visitor.website_id));
-            store.add(this.browse(visitor.id), data);
-        }
```

Comment:
The `visitor` fields that actually exist in just `website` should stay here. The `_to_store` was fine (it should not even be necessary to define it if we updated mock server to be like server, but that's another task).